packages/react-admin/src/mui/list/List.js

```diff
@@ -18,17 +18,26 @@ import Header from '../layout/Header';
 import Title from '../layout/Title';
 import DefaultPagination from './Pagination';
 import DefaultActions from './Actions';
+import DefaultMobileActions from './MobileActions';
 import { crudGetList as crudGetListAction } from '../../actions/dataActions';
 import { changeListParams as changeListParamsAction } from '../../actions/listActions';
 import translate from '../../i18n/translate';
 import removeKey from '../../util/removeKey';
 import defaultTheme from '../defaultTheme';
 import withChildrenAsFunction from '../withChildrenAsFunction';
+import SelectActionButton from './SelectActionButton';
+import SelectActionMenuItem from './SelectActionMenuItem';
 
 const styles = {
     noResults: { padding: 20 },
 };
 
+const DefaultSelectActionButton = props => (
+    <SelectActionButton {...props}>
```

Comment:
This should be in the Actions component directly. If you want to customize actions, it should be done as usual. Bulk actions should not be handled differently